test/mocha/let.js

```diff
@@ -0,0 +1,30 @@
+var Uglify = require('../../');
+var assert = require("assert");
+
+describe("let", function() {
+    it("Should not produce `let` as a variable name in mangle", function(done) {
+        this.timeout(10000);
+
+        // Produce a lot of variables in a function and run it through mangle.
+        var s = '"use strict"; function foo() {';
+        for (var i = 0; i < 21000; ++i) {
```

Comment:
The test looks for the symbols that had existed before and after the erroneously generated `let`. If the mangle algorithm changes, this test would have to be updated.

But if @mishoo or @rvanvelzen would be good with a 50 second `npm test` (2X estimated Travis CI run time) to test all 3 letter mangle var names, then we could do that.